src/tools/pmempool/output.c

```diff
@@ -816,6 +816,18 @@ out_get_arch_machine_str(uint16_t machine)
 	return str_buff;
 }
 
+/*
+ * out_get_finish_state_str -- get a string representation of the finish state
```

Comment:
This comment should be also update after rename the function.
IMHO consider also edit the commit message "pool: add finish state in pmempool_info" still corresponds to "finish state"